src/main/scala/org/globalforestwatch/summarystats/forest_change_diagnostic/ForestChangeDiagnosticAnalysis.scala

```diff
@@ -42,6 +42,9 @@ object ForestChangeDiagnosticAnalysis extends SummaryAnalysis {
     kwargs: Map[String, Any]
   )(implicit spark: SparkSession): RDD[ValidatedLocation[ForestChangeDiagnosticData]] = {
     features.persist(StorageLevel.MEMORY_AND_DISK)
+    // For debugging - will be removed before checkin.
```

Comment:
Looks like you missed removing this

Reply:
Yes, removed all the debugging statements in the latest commit.